ipn/ipnlocal/local.go

```diff
@@ -823,15 +823,16 @@ func (b *LocalBackend) UpdateStatus(sb *ipnstate.StatusBuilder) {
 			ss.UserID = b.netMap.User()
 			if sn := b.netMap.SelfNode; sn.Valid() {
 				peerStatusFromNode(ss, sn)
-				if c := sn.Capabilities(); c.Len() > 0 {
-					ss.Capabilities = c.AsSlice()
-				}
 				if cm := sn.CapMap(); cm.Len() > 0 {
+					ss.Capabilities = make([]tailcfg.NodeCapability, 1, cm.Len()+1)
+					ss.Capabilities[0] = "DEPRECATED; use CapMap; see https://github.com/tailscale/tailscale/issues/11508"
```

Comment:
i meant something like`https://tailscale.com/a/deprecated/node-capabilities` or something so that it conforms the the guide lines at least

Reply:
I'm afraid that'll blend in too much.

And I want a URL people can actually go to and learn more.

I guess I can make the hostname and suffix after `/s/` be SCREAM CAPS and do something like `https://TAILSCALE.COM/s/DEPRECATED-NODE-CAPS#see-https://github.com/tailscale/tailscale/issues/....` and it'll actually redirect too?